app/enterprise/changelog.md

```diff
@@ -4,6 +4,22 @@ no_search: true
 no_version: true
 ---
 
+## 2.5.0.1
+**Release Date** 2021/08/12
+
+### Fixes
+
+#### Enterprise
+- Updates Kong Dev Portal templates' JQuery dependency to v3.6.0, improving security.
+- Now, bootstrap migrations for multi-node Apache Cassandra 4.0.0 clusters work as expected.
```

Comment:
```suggestion
- Now, bootstrap migrations for multi-node Apache Cassandra clusters work as expected.
```

This was actually failing on 3.11.10 clusters as well so I think we can be generic instead.

Reply:
ok, will remove the version